### ISSUE TITLE
Typo on directive

### ISSUE THREAD

**Issue by LeMoussel** [bopoda/robots-txt-parser#16]
I found some typos in robots.txt files. 
For example, for Disallow directive,  I found 
> D i s a l l o w 
> Disllaow
> Desallow
> diallow
> Disalow
> disallaw

The feature would be to log message when there is typo on this directive .
```
	const DIRECTIVE_ALLOW = 'allow';
	const DIRECTIVE_DISALLOW = 'disallow';
	const DIRECTIVE_HOST = 'host';
	const DIRECTIVE_SITEMAP = 'sitemap';
	const DIRECTIVE_USERAGENT = 'user-agent';
	const DIRECTIVE_CRAWL_DELAY = 'crawl-delay';
	const DIRECTIVE_CLEAN_PARAM = 'clean-param';
```

**Comment by bopoda** [comment]
We can Log message.
Еhe expected behavior is that we skip directives with typos?
```
User-Agent: *
Disallow: /path1
Disalow: /path2
```
expected to disallow only /path1?

**Comment by LeMoussel** [comment]
Yes and log message about typo 'Disalow' (for user information only).

**Comment by thingNumber1** [comment]
[Google's guide](https://developers.google.com/webmasters/control-crawl-index/docs/robots_txt?hl=en#file-format) says:
> Handling of <field> elements with simple errors / typos (eg "useragent" instead of "user-agent") is undefined and may be interpreted as correct directives by some user-agents.

I think the beast approach is to add a (non-default) directive to support it.

If you agree with me, there are 2 (at least) approach:

- To create a list of common mistakes.
- Or, the one I like, to calculate the [levenshtein distance](http://php.net/manual/es/function.levenshtein.php) of the unknown field elements (possible typos) against the the list of known element (well written) and support then as their corrected version if they are close enought.

**Comment by LeMoussel** [comment]
👍  But But it would be interesting to know all simple errors / typos when parsing robots.txt file. Store it in list ?

**Comment by bopoda** [comment]
But are you sure that it is a good idea to add typos logs to this repo?
I asked because it will be more code, more logic, which can be not necessary in general. 
It can give more junk code.

**Comment by thingNumber1** [comment]
Personally, I would take the levenshtein distance approach and give the developers the option to pass a variable to get the errors and the can store it or not or do what ever they think is best, but I wouldn't create log within the class...
